LottieDropper/Dropbox/DropboxBrowserViewModel.m

```diff
@@ -46,29 +49,43 @@ - (void)initializeDropboxClient: (void (^)(void)) done  {
 		NSLog(@" No Dropbox user client");
 		return;
 	}
-	[[self.client.filesRoutes listFolder:@""]
```

Comment:
Dit is een oplossing maar was wel niet de vraag. De vraag was. 
1. Eerst toon je de lokale file
2. Als er verschil is met de file die van de server komt toon je de file opnieuw.
3. Als er geen verschil is toon je hem niet.

Ook connectivity geeft je geen zekerheid over een *internet* connectie. Het enige wat het zegt is dat er een netwerk is. Het kan nog steeds dat er op dat netwerk geen connectie met dropbox gemaakt kan worden.